questions.md

```diff
@@ -0,0 +1 @@
+- Will pull request have a merge conflict if the commits on master is ahead of pull request's commits? (master has updated commits while PR branch is behind master)
```

Comment:
That depends on the commits. For example, if both branches edit the same line of the same file, there will be conflicts. However, conflicts are not automatic just because the master branch is ahead.

Reply:
Thank you! :)